Util/Tokenization/src/test/resources/org/tribuo/util/tokens/impl/test/regression-text_bert-base-uncased_fails.txt

```diff
@@ -0,0 +1,91 @@
+ابو صیر بنا (به عربی: أبو صیر بنا) یک روستا و محوطه باستانی در مصر است که در استان غربیه واقع شده‌است. 	ا	##ب	##و	ص	##ی	##ر	ب	##ن	##ا	(	ب	##ه	ع	##ر	##ب	##ی	:	ا	##ب	##و	ص	##ی	##ر	ب	##ن	##ا	)	ی	##ک	ر	##و	##س	##ت	##ا	و	م	##ح	##و	##ط	##ه	ب	##ا	##س	##ت	##ان	##ی	د	##ر	م	##ص	##ر	ا	##س	##ت	ک	##ه	د	##ر	ا	##س	##ت	##ان	غ	##ر	##ب	##ی	##ه	و	##ا	##ق	##ع	[UNK]	.

```

Comment:
This file isn't referenced by any test.

Reply:
It is now referenced by the javadoc which documents that these correspond to still-failing regression tests.  per a previous conversation -it might be nice to have those if/when someone decides they want to tighten up the discrepancies between our impl and the reference python version.